include/pokedex_screen.h

```diff
@@ -11,6 +11,11 @@
 #define DEX_CATEGORY_URBAN         7
 #define DEX_CATEGORY_RARE          8
 
+#define DEX_CATEGORY_SIZE          (sizeof(gDexCategories)/sizeof(struct {                                      \
+                                                                      const struct PokedexCategoryPage * page;  \
+                                                                      u8 count;                                 \
+                                                                  }))
+
```

Comment:
No need for this. `NELEMS(gDexCategories)`

Reply:
Oops, forgot that existed

Reply:
Sorry should've been clearer, the usage of `DEX_CATEGORY_SIZE` itself can just be replaced with `NELEMS(gDexCategories)`.